spec/controllers/curation_concerns/generic_works_controller_spec.rb

```diff
@@ -77,6 +78,64 @@
       end
     end
 
+    context 'with unsuppressed work' do
+      let(:work) { create(:public_generic_work) }
+      let(:document) { SolrDocument.new(work.to_solr) }
+      let(:presenter) { CurationConcerns::WorkShowPresenter.new(document, nil) }
+      before do
+        allow_any_instance_of(GenericWork).to receive(:suppressed?).and_return(false)
```

Comment:
Why is there an instance of a GenericWork here? Shouldn't we just be dealing in the realm of Solr?